RestaurantOrderingSystem/src/main/java/utils/DBManager.java

```diff
@@ -121,13 +121,13 @@ public static User getLoggedInUser() {
 		return getUser(authentication.getName());
 	}
 
-	public static Order getOrder() {
+	public static Order getActiveOrder() {
 		SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 		
 		Session session = sessionFactory.openSession();	
 		session.beginTransaction();
 		
-		Query<Order> query=  session.createQuery("from Order ");
+		Query<Order> query=  session.createQuery("from Order where orderStatus = 'ACTIVE'");
```

Comment:
this will only work if there is only one active order in the system for all users.  We should probably access the active order from the users order history directly